Specs/DateTime/Italian/DateExtractor.json

```diff
@@ -131,29 +120,7 @@
 }
 ,
 {
-  "Input": "I'll go back on 4.22",
```

Comment:
Try to keep as many of the original test cases if you can.
Only remove if they really don't make sense in Italian, and there is no other similar cases that could be used.